addons/web/static/src/search/comparison_menu/comparison_menu.xml

```diff
@@ -8,9 +8,10 @@
                 <span class="o_dropdown_title">Comparison</span>
             </t>
             <t t-foreach="items" t-as="item" t-key="item.id">
-                <DropdownItem
+                <SearchDropdownItem
```

Comment:
do not change DropdownItem name in template, instead link to correct component in JS